### PR TITLE
Port policy from WASI to waPC

### LINKED ISSUE #2: Port the policy to use waPC instead of WASI
The policy should be ported to the new Chimera policy architecture

### PR DESCRIPTION
**Pull request by flavio**
Move to the new policy format, fixes #2

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+use serde::{Deserialize, Serialize};
+use serde_json::Result;
+
+mod settings;
+use settings::Settings;
+
+mod taint;
+use taint::Taint;
+
+#[derive(Deserialize, Serialize, Debug)]
```
Comment (ereslibre) [inline src/lib.rs:15]:
```suggestion
#[derive(Serialize, Debug)]
```

File: src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+struct ValidationResponse {
+    accepted: bool,
+    message: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
```
Comment (ereslibre) [inline src/lib.rs:21]:
```suggestion
#[derive(Deserialize, Debug)]
```

**Review by ereslibre — APPROVED** [review]
LGTM. I don't have better suggestions right now, I think we can improve later.

File: src/lib.rs (hunk 1 above)
Comment (flavio) [reply src/lib.rs:15]: The `Deserialize` trait is needed to run the unit tests

File: src/lib.rs (hunk 2 above)
Comment (flavio) [reply src/lib.rs:21]: The `Deserialize` trait is needed to run the unit tests

**Pull request merged**